src/integration/integration_api_scheduler_test.go

```diff
@@ -30,7 +32,9 @@ var _ = Describe("Integration_Api_Scheduler", func() {
 		httpClient.Transport.(*http.Transport).TLSClientConfig = apiTLSConfig
 		httpClient.Timeout = apiSchedulerHttpRequestTimeout
 		apiServerConfPath = components.PrepareApiServerConfig(components.Ports[APIServer], dbUrl, fmt.Sprintf("https://127.0.0.1:%d", components.Ports[Scheduler]), tmpDir)
+		schedulerConfPath = components.PrepareSchedulerConfig(dbUrl, fmt.Sprintf("https://127.0.0.1:%d", components.Ports[ScalingEngine]), tmpDir)
 		startApiServer()
+		schedulerProcess = startScheduler()
```

Comment:
Do we need to start/stop the scheduler for each test or is it sufficient to start/stop it once for all tests?